api/interservice/infra_proxy/response/databags.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package chef.automate.domain.infra_proxy.response;
+option go_package = "github.com/chef/automate/api/interservice/infra_proxy/response";
+
+message DataBags {
+    // List of data bags item.
+    repeated DataBagListItem data_bags = 2;
+}
+
+message DataBagListItem {
+    // Name of the data bag item.
+    string name = 1;
+}
+
+message DataBag {
+    // Stringified json of the data bag item.
```

Comment:
thanks for the comments in the proto! that'll help us get there quicker when we're ready for api docs 🎉